tests/test_aligner.yml

```diff
@@ -48,40 +81,85 @@
     - preprocessing
   files:
     - path: results/csv/markduplicates.csv
+      md5sum: 0d6120bb99e92f6810343270711ca53e
     - path: results/csv/markduplicates_no_table.csv
+      md5sum: 2a2d3d4842befd4def39156463859ee3
     - path: results/csv/recalibrated.csv
+      md5sum: 42628ec994c16f565e5407b40a9c1ac3
     - path: results/multiqc
     - path: results/preprocessing/markduplicates/test/test.md.cram
+    # binary changing on reruns
     - path: results/preprocessing/markduplicates/test/test.md.cram.crai
+    # binary changing on reruns
     - path: results/preprocessing/recal_table/test/test.recal.table
+      md5sum: 75ba4376a17ca69c5134153302f82e92
     - path: results/preprocessing/recalibrated/test/test.recal.cram
+    # binary changing on reruns
     - path: results/preprocessing/recalibrated/test/test.recal.cram.crai
+    # binary changing on reruns
     - path: results/reference/dbsnp/dbsnp_146.hg38.vcf.gz.tbi
+      md5sum: 628232d0c870f2dbf73c3e81aff7b4b4
     - path: results/reference/dict/genome.dict
+      md5sum: 2433fe2ba31257337bf4c4bd4cb8da15
     - path: results/reference/dragmap/hash_table.cfg
+      # hash_table.cfg contains many strings which we could test for - which do we want to test?
```

Comment:
```suggestion
      # hash_table.cfg test for strings
```

Reply:
The content of ´hash_table.cfg` is as shown below. (I wanted to attach the file to this post, but GitHub didn't agree. Hence the big ugly splash.) I guess the file contains too many values for them all to be tested. Are there any particular values which should be tested?
```
# Automatically generated by build_hash_table (local version).
#    Command line: dragen-os --build-hash-table true --ht-reference genome.fasta --output-directory dragmap --ht-num-threads 2
#    Hash table version 8
#
# Do not modify.

reference_source     = 'genome.fasta'
alt_liftover         = ''
reference_name       = '/home/aspe/dev/fork_sarek/work/2c/d70b7e11d7ae1d492e46697a2c33f4/dragmap/reference.bin'
reference_index      = '/home/aspe/dev/fork_sarek/work/2c/d70b7e11d7ae1d492e46697a2c33f4/dragmap/ref_index.bin'
reference_sequences  = 1
reference_len        = 368640
reference_len_raw    = 40001
reference_len_not_n  = 40001
reference_alt_seed   = 204800
reference_alt_start  = 204800
hash_table           = '/home/aspe/dev/fork_sarek/work/2c/d70b7e11d7ae1d492e46697a2c33f4/dragmap/hash_table.bin'
extend_table         = '/home/aspe/dev/fork_sarek/work/2c/d70b7e11d7ae1d492e46697a2c33f4/dragmap/extend_table.bin'
hash_table_bytes     = 884736
extend_table_records = 32
digest_type          = 1
digest               = 0xF13426C0
ref_digest           = 0x00000000
ref_index_digest     = 0x00000000
hash_digest          = 0xA2A09BAA
liftover_digest      = 0x00000000
extend_table_digest  = 0x4E938B16
pri_seed_bases       = 19
max_seed_bases       = 147
max_ext_increment    = 12
ref_seed_interval    = 1
table_addr_bits      = 20
table_size_64ths     = 54
max_seed_freq        = 16
pri_max_seed_freq    = 16
max_seed_freq_len    = 98
target_seed_freq     = 4
min_freq_to_extend   = 17
thinning_freq_cap    = 12
max_thinning_factor  = 1
pri_crc_bits         = 39
sec_crc_bits         = 39
seed_len_cost        = 1
seed_freq_cost       = 0.5
extension_cost       = 0
ext_step_cost        = 0.7
ext_rec_cost         = 4
repair_strategy      = 0
min_repair_prob      = 0.2
anchor_bin_bits      = 0
hi_freq_rand_hit     = 1
ext_rand_hit_freq    = 8
pri_crc_poly         = 68B22EEA95
sec_crc_poly         = 68B22EEA95
reference_sequence0     = 'chr22'
reference_start0        = 163840
reference_beg_trim0     = 0
reference_end_trim0     = 0
reference_len0          = 40001
```